src/app/zap-templates/zcl/data-model/chip/laundry-washer-cluster.xml

```diff
@@ -0,0 +1,91 @@
+    <attribute side="server" code="0x0001" define="STANDARD_NAMESPACE" type="ENUM16"                                        writable="false" optional="false"  isNullable="true">StandardNamespace</attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_MODES" type="ARRAY" entryType="ModeOptionStruct"               writable="false" optional="false" isNullable="false">SupportedModes</attribute>
+    <attribute side="server" code="0x0003" define="CURRENT_MODE"    type="INT8U"                                            writable="false" optional="false" isNullable="false" reportable="true">CurrentMode</attribute>
+    <attribute side="server" code="0x0004" define="START_UP_MODE"   type="INT8U"                                            writable="true" optional="true"  isNullable="true">StartUpMode</attribute>
+    <attribute side="server" code="0x0005" define="ON_MODE"         type="INT8U"                                            writable="true"  optional="true" isNullable="true">OnMode</attribute>
+
+    <!-- Commands -->
+    <command source="client" code="0x00" name="ChangeToMode" optional="false">
+      <description>
+        If the NewMode field doesn't match the Mode field in an entry of the SupportedModes list the server SHALL respond with an INVALID_COMMAND status response.
+        If the NewMode field matches the Mode field in an entry of the SupportedModes list but the device is unable to transition as requested, the server SHALL respond with a FAILURE status response.
+        If the NewMode field matches the Mode field in an entry of the SupportedModes list and the device is able to transition as requested the server SHALL set the CurrentMode attribute to the NewMode value and SHALL respond with a SUCCESS status response.
+      </description>
+      <arg name="NewMode" type="INT8U" optional="false"/>
+    </command>
+
+    <command source="client" code="0x01" name="ChangeToModeWithStatus" optional="true">
+      <description>
+        This command is used to change device modes using the same mechanism and semantics as ChangeToMode, and additionally obtaining a response with an ability for clients to determine causes of failures with fine-grained details. For status response depending on NewMode field, see ChangeToMode command.
+        On receipt of this command the device SHALL respond with a ChangeToModeResponse command.
+      </description>
+      <arg name="NewMode" type="INT8U" optional="false"/>
+    </command>
+
+    <command source="server" code="0x02" name="ChangeToModeResponse" optional="true">
+      <description>
+        This command is sent by the device on receipt of the ChangeToModeWithStatus command.
+      </description>
+      <arg name="Status" type="ENUM8" optional="false"/>
+      <arg name="StatusText" type="CHAR_STRING" optional="true"/>
+    </command>
+
+  </cluster>
+
+  <bitmap name="ModeSelectFeature" type="BITMAP32">
```

Comment:
For a bunch of these other clusters, we can use single definitions of the enums/bitmaps/structs in mode-select-cluster.xml and just add all the relevant cluster ids to them.